packages/amplication-client/src/VersionControl/Commit.tsx

```diff
@@ -88,59 +136,182 @@ const Commit = ({
+          validateOnMount
+          innerRef={formikRef}
+        >
+          {(formik) => {
+            const handlers = {
+              SUBMIT: formik.submitForm,
+            };
+
+            return (
+              <Form>
+                {!commitChangesLoading && (
+                  <GlobalHotKeys keyMap={keyMap} handlers={handlers} />
+                )}
+                {showCommitMessage && (
+                  <TextField
+                    rows={3}
+                    textarea
+                    name="message"
+                    label={noChanges ? "Build message" : "Commit message..."}
+                    disabled={commitChangesLoading}
+                    autoFocus
+                    hideLabel
+                    placeholder={
+                      noChanges ? "Build message" : "Commit message..."
+                    }
+                    autoComplete="off"
+                  />
+                )}
+                {!dotNetGeneratorEnabled && (
+                  <MultiStateToggle
+                    className={`${CLASS_NAME}__technology-toggle`}
+                    label=""
+                    name="action_"
+                    options={OPTIONS}
+                    onChange={handleOnSelectLanguageChange}
+                    selectedValue={"node"}
+                  />
+                )}
+                {commitBtnType === CommitBtnType.Button ? (
+                  <div>
+                    <FlexItem
+                      direction={EnumFlexDirection.Row}
+                      itemsAlign={EnumItemsAlign.Center}
+                      gap={EnumGapSize.None}
+                    >
+                      <CommitButton
+                        commitBtnType={commitBtnType}
+                        commitMessage={formikRef.current?.values?.message}
+                        onCommitChanges={handleCommitBtnClicked}
+                        commitStrategy={
+                          formikRef.current?.values?.commitStrategy
+                        }
+                      ></CommitButton>
+                      <SelectMenu
+                        title=""
+                        icon="chevron_down"
+                        buttonStyle={EnumButtonStyle.Text}
+                        className={`${CLASS_NAME}__commit-strategy`}
+                      >
+                        <SelectMenuModal align="right" withCaret>
+                          <SelectMenuList>
+                            {COMMIT_STRATEGY_OPTIONS.map((item, index) => (
+                              <CreateCommitStrategyButtonItem
+                                key={index}
+                                item={item}
+                                hasPendingChanges={pendingChanges?.length > 0}
+                                onCommitStrategySelected={
+                                  handleOnSelectCommitStrategyChange
+                                }
+                              ></CreateCommitStrategyButtonItem>
```

Comment:
```suggestion
                              />
```

Reply:
@mulygottlieb  How? This component has props